kayak_core/src/context.rs

```diff
@@ -1,18 +1,25 @@
 use flo_binding::Changeable;
+use morphorm::Hierarchy;
 use std::collections::HashMap;
 
-use crate::{widget_manager::WidgetManager, Event, EventType, Index, InputEvent};
+use crate::{
+    multi_state::MultiState, widget_manager::WidgetManager, Event, EventType, Index, InputEvent,
+};
 
 pub struct KayakContext {
     widget_states: HashMap<crate::Index, resources::Resources>,
     global_bindings: HashMap<crate::Index, Vec<flo_binding::Uuid>>,
     widget_state_lifetimes:
         HashMap<crate::Index, HashMap<flo_binding::Uuid, Box<dyn crate::Releasable>>>,
-    current_id: Index,
+    pub current_id: Index,
```

Comment:
Should this actually be public? The `set_current_id` has a bit of extra logic that could be bypassed with this.

https://github.com/StarArawn/kayak_ui/blob/772a6d79c7d7176acb3012258fd39441b183ab0b/kayak_core/src/context.rs#L102-L106

Reply:
I exposed for testing, but I'll revert that change.